src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
```

Comment:
Prefer using `const` for the `calculateRentalCost` function declaration since you do not intend to reassign it.